packages/spindle-ui/src/Pagination/PaginationNavigation.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FC } from 'react';
+import ChevronRightBold from '../Icon/ChevronRightBold';
+import ChevronLeftBold from '../Icon/ChevronLeftBold';
+import TriangleendRightBold from '../Icon/TriangleendRightBold';
+import TriangleendLeftBold from '../Icon/TriangleendLeftBold';
+
+type Props = {
+  type: string;
```

Comment:
```suggestion
  type: 'first' | 'last' | 'next' | 'prev';
```
nits. リテラルにできそうです！このコンポーネント内部でもswitchがType Guardとなり推論が改善されそうです